leveller.js

```diff
@@ -61,16 +61,24 @@
     var diff = targetHeight - currentHeight;
     if (diff === 0) return;
 
+    var isHeightProperty = (this.options.cssProperty.indexOf('eight') > 0);
     var $styleElement = this.options.cssSelector ? $element.find(this.options.cssSelector) : $element;
-    var styleValue = targetHeight;
-    if (this.options.cssProperty.indexOf('eight') < 0) {
+    var styleValue;
+
+    if (isHeightProperty && this.options.cssSelector) {
+      styleValue = $styleElement[this.options.heightMethod]() + diff;
```

Comment:
WRT `this.options.cssSelector`, are you checking here for its existence? I suppose there is no valid CSS selector that would be falsy in JS land, but again, checking clarity of intent here :)